### PR TITLE
Use io.TextIOWrapper to prevent "x85" issue creating None

### LINKED ISSUE #76: TypeError with bulk_import with CSV if a column has "x85"
From: https://github.com/treasure-data/pytd/issues/52

When we have `x85` in a column, bulk import fails with `TypeError`

```py
>>> import pandas as pd
>>> import tdclient
>>> td = tdclient.Client()
>>>
>>> df = pd.DataFrame(data={'col1': [1, 2], 'col2': [3, 10], 'col3': ['foo', "\x85."]})
>>> import time
>>> session_name = "session-{}".format(int(time.time()))
>>> fname = "foo.csv"
>>> df.to_csv(fname)
>>> data = open(fname, 'rb').read()
>>> data
b',col1,col2,col3\n0,1,3,foo\n1,2,10,\xc2\x85.\n'
>>> bulk_import = td.create_bulk_import(session_name, "aki", "test")
>>> bulk_import.upload_file("part", "csv", fname)
/Users/aki/src/pytd/.venv/lib/python3.7/site-packages/tdclient/api.py:460: RuntimeWarning: records should have "time" column to import records properly.
  warnings.warn("records should have \"time\" column to import records properly.", category=RuntimeWarning)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/aki/src/pytd/.venv/lib/python3.7/site-packages/tdclient/bulk_import_model.py", line 190, in upload_file
    response = self._client.bulk_import_upload_file(self.name, part_name, fmt, file_like)
  File "/Users/aki/src/pytd/.venv/lib/python3.7/site-packages/tdclient/client.py", line 494, in bulk_import_upload_file
    return self.api.bulk_import_upload_file(name, part_name, format, file)
  File "/Users/aki/src/pytd/.venv/lib/python3.7/site-packages/tdclient/bulk_import_api.py", line 118, in bulk_import_upload_file
    with contextlib.closing(self._prepare_file(file, format, **kwargs)) as fp:
  File "/Users/aki/src/pytd/.venv/lib/python3.7/site-packages/tdclient/api.py", line 428, in _prepare_file
    for item in items:
  File "/Users/aki/src/pytd/.venv/lib/python3.7/site-packages/tdclient/api.py", line 507, in _read_csv_file
    record = dict([ (k, value(v)) for (k, v) in row.items() ])
  File "/Users/aki/src/pytd/.venv/lib/python3.7/site-packages/tdclient/api.py", line 507, in <listcomp>
    record = dict([ (k, value(v)) for (k, v) in row.items() ])
  File "/Users/aki/src/pytd/.venv/lib/python3.7/site-packages/tdclient/api.py", line 491, in value
    return int(s)
TypeError: int() argument must be a string, a bytes-like object or a number, not 'NoneType'
```

Since `x85` is [next line](https://www.fileformat.info/info/unicode/char/0085/index.htm), `codecs` creates next line.

```py
>>> import io
>>> import codecs
>>> data = b',col1,col2,col3\n0,1,3,foo\n1,2,10,\xc2\x85.\n'
>>> list(codecs.getreader('UTF-8')(io.BytesIO(data)))
[',col1,col2,col3\n', '0,1,3,foo\n', '1,2,10,\x85', '.\n']
```

This doesn't occur with `io.TextIOWrapper`, so should be replaced with it.

```py
>>> list(io.TextIOWrapper(io.BytesIO(data), 'UTF-8'))
[',col1,col2,col3\n', '0,1,3,foo\n', '1,2,10,\x85.\n']
```

### PR DESCRIPTION
**Pull request by chezou**
Fixes #76 

```py
n [1]: import pandas as pd

In [2]: import tdclient

In [3]: td = tdclient.Client()

In [4]: df = pd.DataFrame(data={'col1': [1, 2], 'col2': [3, 10], 'col3': ['foo', "\x85."]})

In [5]: fname = "/tmp/foo.csv"

In [6]: df.to_csv(fname)

In [7]: import time

In [8]: session_name = f"session-{int(time.time())}"

In [10]: td.create_log_table("aki", "test")
Out[10]: True

In [23]: df['time'] = int(time.time())

In [24]: df.to_csv(fname)

In [25]: session_name = f"session-{int(time.time())}"

In [26]: bulk_import = td.create_bulk_import(session_name, "aki", "test")

In [27]: bulk_import.freeze()
Out[27]: True

In [29]: bulk_import.delete()
Out[29]: True

In [30]: session_name = f"session-{int(time.time())}"

In [31]: bulk_import = td.create_bulk_import(session_name, "aki", "test")

In [32]: bulk_import.upload_file("part", "csv", fname)

In [33]: bulk_import.freeze()
Out[33]: True

In [35]: bulk_import.perform(wait=True)
Out[35]: <tdclient.job_model.Job at 0x1154ae668>

In [36]: bulk_import.error_records
Out[36]: 0

In [37]: bulk_import.valid_records
Out[37]: 2

In [38]: bulk_import.commit(wait=True)
Out[38]: True

In [39]: bulk_import.delete()
Out[39]: True

In [42]: job = td.query("aki", "select * from test", type="presto")

In [46]: job.wait()

In [49]: list(job.result())
Out[49]: [[1, 3, 'foo', 1570499820], [2, 10, '\x85.', 1570499820]]
```

### CODE REVIEW

File: tdclient/api.py [hunk 1]
```diff
@@ -617,10 +617,6 @@ def _read_json_file(self, file_like, **kwargs):
     def _read_csv_file(
         self, file_like, dialect=csv.excel, columns=None, encoding="utf-8", **kwargs
     ):
-        def getreader(file_like):
-            for s in codecs.getreader(encoding)(file_like):
```
Comment (yyuu) [inline tdclient/api.py]: Let's remove import line for `codecs`

File: tdclient/api.py [hunk 2]
```diff
@@ -638,13 +634,13 @@ def value(s):
                 return s
 
         if columns is None:
-            reader = csv.DictReader(getreader(file_like), dialect=dialect)
+            reader = csv.DictReader(io.TextIOWrapper(file_like, encoding), dialect=dialect)
```
Comment (yyuu) [inline tdclient/api.py]: It seems there are similar code in test's helper methods. It'd be better changing these codes as well in this time.

https://github.com/treasure-data/td-client-python/blob/770e2e104631b27c05d075ecabc9d2a180c71e15/tdclient/test/test_helper.py#L114

File: tdclient/api.py (hunk 1 above)
Comment (chezou) [reply tdclient/api.py]: removed 
1c6f93d

File: tdclient/api.py (hunk 2 above)
Comment (chezou) [reply tdclient/api.py]: Removed as well 1c6f93d 120f791

**Pull request merged**